Source/Analytics/Events.Reporting/DataCollectors/DataCollectorRegistered.cs

```diff
@@ -1,47 +1,57 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) The International Federation of Red Cross and Red Crescent Societies. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
 using System;
 using Dolittle.Events;
 
 namespace Events.Reporting.DataCollectors
 {
     public class DataCollectorRegistered : IEvent
     {
-        public Guid DataCollectorId { get; }
+        public DataCollectorRegistered (
+                string fullName,
+                string displayName,            
+                int yearOfBirth,
+                int sex,
+                int preferredLanguage,
+                double locationLongitude, 
+                double locationLatitude,
+                DateTimeOffset registeredAt,
+                string region,
+                string district,
+                Guid dataVerifierId
+            )
+        {
+            this.FullName = fullName;
+            this.DisplayName = displayName;
+            this.YearOfBirth = yearOfBirth;
+            this.Sex = sex;
+            this.PreferredLanguage = preferredLanguage;
+            this.LocationLongitude = locationLongitude;
+            this.LocationLatitude = locationLatitude;
+            this.RegisteredAt = registeredAt;
+            this.Region = region;
+            this.District = district;
+            this.DataVerifierId = dataVerifierId;
+        }
+    
         public string FullName { get; }
         public string DisplayName { get; }
         public int YearOfBirth { get; }
-        public int Sex { get; }
+        public int Sex { get;}
         public int PreferredLanguage { get; }
         public double LocationLongitude { get; }
         public double LocationLatitude { get; }
 
         public string Region { get; }
         public string District { get; }
+        
+        public Guid DataVerifierId {get; } 
```

Comment:
Is it correct that it is the dataverifierid here? should it noe still be datacollectorid?

Reply:
@woksin agree that this is correct? this needs to look exactly like the event defined in reporting, right?

Reply:
I think it should be a DataVerifierId – In Reporting's `DataCollectorRegistered`, a `guid DataVerifierId` is included but not a `guid DataCollectorId`. The `DataCollectorId` should come from the EventSourceId. And the events should match, as you say.

Reply:
It's correct that the data collector ID comes from the EventSourceId.

Structure wise, an event from another BC does not need have all the properties of the original. You could just include the properties that you are interested in. But they would have to be the same type and have the same name, as it is here